compiler/test/runnable/cdvecfill.sh

```diff
@@ -10,17 +10,17 @@ obj_file=${RESULTS_TEST_DIR}/${TEST_NAME}.o
 
 if [ $OS == "linux" ] && [ $MODEL == "64" ]; then
   $DMD -betterC -c -O -m64 ${src_file} -of${obj_file}
-  objdump --disassemble --disassembler-options=intel-mnemonic ${obj_file} | tail -n+3 | sed 's/[ \t]\s*$//' > ${tmp_file}
+  objdump --disassemble --disassembler-options=intel ${obj_file} | tail -n+3 | sed 's/[ \t]\s*$//' > ${tmp_file}
```

Comment:
For some reason `intel-mnemonic` still generated AT&T syntax for me but just `intel` worked as expected.